extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -416,7 +416,10 @@ export const SendAmount = ({
             <div className="SendAmount__btn-continue">
               <Button
                 disabled={
-                  loadingRate || formik.values.amount === "0" || !formik.isValid
+                  loadingRate ||
+                  formik.values.amount === "0" ||
+                  !formik.isValid ||
+                  !destinationAmount
```

Comment:
Just noticed this bug: in a path payment, if there's no path available (for ex: using SRT in testnet), it still allows you to continue. Let's disable that